pdfmaker/css/torDOTcom/novel.scss

```diff
@@ -56,6 +57,8 @@ $sizefootnote: 9pt;
 $sizecaption: $smaller;
 $sizefigsrc: 7pt;
 $sizesuperscript: 6pt;
+$sizehwritingprint: $gridheight * .8;
+$sizehwritingnormal: $gridheight * .69;
```

Comment:
OK, that's what I had but then reviewed your new README for the repo and saw #4 under "add a new global style:" ;)  I'll revert it to actual sizes